docs/sources/userguide/dockervolumes.md

```diff
@@ -141,6 +146,14 @@ be deleted.  To delete the volume from disk, you must explicitly call
 `docker rm -v` against the last container with a reference to the volume. This
 allows you to upgrade, or effectively migrate data volumes between containers.
 
+> **Note:** Docker will not warn you when removing a container *without* 
+> providing the `-v` option to delete its volumes. If you remove containers
+> without using the `-v` option, you may end up with "orphaned" volumes; 
+> volumes that are no longer referenced by a container.
+> Orphaned volumes are difficult to get rid off and can take up a large amount
+> of disk space. We're working on improving volume management and you can check
+> progress on this in [pull request #8484](https://github.com/docker/docker/pull/8484)
```

Comment:
I considered adding a link to @cpuguy83 s "docker-volumes", which is basically a POC external tool that implements the PR mentioned (https://github.com/cpuguy83/docker-volumes/). I wasn't sure if linking to a "third-party" tool was appropriate?

The other workaround is to manually purge volumes in the `/var/lib/docker/vfs/dir/` directory, which is not really best practice.

Should I include a link to docker-volumes?